index.js

```diff
@@ -20,11 +20,11 @@ module.exports = {
 		return nodeNightlyVer.then(latest => {
 			const conf = new Configstore(pkg.name);
 			conf.set('version', latest);
-			const os = process.platform;
+			const os = process.platform  === 'win32' ? 'win' : process.platform;
 			const arch = process.arch;
 			const type = 'nightly';
 			osArchString = `${latest}-${os}-${arch}`;
-			const url = `https://nodejs.org/download/${type}/${latest}/node-${osArchString}.tar.gz`;
+			const url = `https://nodejs.org/download/${type}/${latest}/node-${osArchString}.zip`;
```

Comment:
I think that the extension needs to be dependant on the platform. I'm pretty sure that `zip` should only be used for windows.

But this is really close!